portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,284 @@
+        <p> I was born in Irvine, California. However, when I was about 
+            two months old my family moved to Beijing, and I spent
+            the next 12 years growing up there. </p>
+        <p> In middle school, I moved back to 
+            Irvine to continue school. I really enjoyed getting to experience both
+            cultures first-hand as I believe I have gained perspective from this
+            transition. </p>
+    </div>
+    <div class="col-lg-6">
+        <img src="images/Beijing.jpg" class="img-fluid">
+    </div>
+</div>
+</div>
+
+<!--- Academics Background -->
+<div class="container-fluid padding">
+<div class="row padding">
+    <div class="col-lg-6">
+        <img src="images/alipay.jpg" class="img-fluid">
+    </div>
+    <div class="col-lg-6">
+        <h2>Academic background</h2>
+        <p> In the past, I've interned for both tech roles and finance roles, and I am
+            currently trying to figure out a solid career path to me to truly enjoy
+            what I do daily. In working for a startup called Obviously AI as an AI 
+            mining intern, I felt immense gratification seeing the python scripts be 
+            used to guage the performance of the company.</p>
+        <p> And as a consultant for Alipay, it was eye-opening to see the immediate 
+            effects of planning a successful promotional event. In this STEP internship 
+            I hope to immerse myself in the tech space and see what being a full-time 
+            SWE would entail.</p>
+    </div>
+</div>
+</div>
+
+
+  
+<!--- Projects -->
+<a id="projects"></a>
+<div class="container-fluid padding">
+<div class="row welcome text-center">
+    <div class="col-12">
+        <h1 class="display-4">Projects</h1>
+    </div>
+    <hr>
+</div>
+</div>
+
+<!--- Project section -->
+<div class="container-fluid padding">
+<div class="row padding">
+    <div class="col-md-6">
+        <div class="card">
+            <img class="card-img-top" src="images/Stars.jpg">
+            <div class="card-body">
+                <h4 class="card-title">UCI Research</h4>
+                <p class="card-text">A project invovling analyzing rotational velocities
+                    of star bodies to model dark matter.</p>
+                <a href="projects.html" class="btn btn-outline-secondary">More details</a>
+            </div>
+        </div>
+    </div>
+    <div class="col-md-6">
+        <div class="card">
+            <img class="card-img-top" src="images/wildfire.jpg">
+            <div class="card-body">
+                <h4 class="card-title">UCB Research</h4>
+                <p class="card-text">A project invovling using small devices
+                     to detect fire as early as possible and request 
+                    satellite images.</p>
+                <a href="projects.html" class="btn btn-outline-secondary">More details</a>
+            </div>
+        </div>
+    </div>
+</div>
+</div>
+
+
+<!--- Hobbies header -->
```

Comment:
Are you using this and the portfolio/src/main/webapp/hobbies.html? Maybe remove whichever one you aren't using?

Reply:
deleted!